updater/updater.go

```diff
@@ -159,7 +160,8 @@ func (u *RepoUpdater) updateBranch(ctx context.Context, log logrus.FieldLogger,
 	for _, dep := range ungrouped {
 		ok, err := u.singleUpdate(ctx, log, branch, dep)
 		if err != nil {
-			return err
+			logrus.WithField("path", dep.Path).WithError(err).Error("error processing update group")
```

Comment:
```suggestion
			logrus.WithField("path", dep.Path).WithError(err).Error("error processing update")
```

Bad paste, not a group.